<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1801</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1402</width>
      <height>1002</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>60</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Weight coeffitients</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Alignement :</string>
     </property>
    </widget>
    <widget class="QSlider" name="alignmentSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>310</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Cohesion :</string>
     </property>
    </widget>
    <widget class="QSlider" name="cohesionSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>310</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Separation :</string>
     </property>
    </widget>
    <widget class="QSlider" name="separationSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>310</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>410</y>
      <width>310</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>360</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Global speed coeffitient :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>460</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighbour radius :</string>
    </property>
   </widget>
   <widget class="QSlider" name="radiusSlider">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>510</y>
      <width>310</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
    <property name="sliderPosition">
     <number>80</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addPredator">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>670</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add predator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSmallObstacle">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>730</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add small obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddBigObstacle">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>790</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add big obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delObstacles">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>850</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete obstacles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delPredators">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>910</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete predators</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFlock">
    <property name="geometry">
     <rect>
      <x>1460</x>
      <y>610</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add 20 flock</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
